eland/operations.py

```diff
@@ -109,10 +109,10 @@ def sum(self, query_compiler, numeric_only=True):
         return self._metric_aggs(query_compiler, 'sum', numeric_only=numeric_only)
 
     def max(self, query_compiler, numeric_only=True):
```

Comment:
From the above frames this `keep_domain` change looks pretty good, would also like @stevedodson to confirm here.